packages/nitro-protocol/test/contracts/TrivalApp/validTransition.test.ts

```diff
@@ -0,0 +1,33 @@
+// @ts-ignore
+import {expectRevert} from '@statechannels/devtools';
+import {Contract} from 'ethers';
+import {HashZero} from 'ethers/constants';
+import TrivialAppArtifact from '../../../build/contracts/TrivialApp.json';
+import {VariablePart} from '../../../src/contract/state.js';
+import {getTestProvider, setupContracts} from '../../test-helpers';
+
+const provider = getTestProvider();
+let trivialApp: Contract;
+
+beforeAll(async () => {
+  trivialApp = await setupContracts(provider, TrivialAppArtifact);
+});
+
+describe('validTransition', () => {
+  it.each`
+    isValid  | description
+    ${true}  | ${'IsValid True = True'}
+    ${false} | ${'IsValid False = True'}
+  `('$description', async ({isValid}: {isValid: boolean}) => {
+    const variablePart: VariablePart = {
+      outcome: HashZero,
+      appData: HashZero,
+    };
+    const isValidFromCall = await trivialApp.validTransition(variablePart, variablePart, 0, 0);
```

Comment:
The existing test has named `from` and `to` `variablePart` structs, which was a bit more readable. Also, here you are testing a pretty special case of transitioning from some fiducial state to itself. That seems overly specific, to me (see other comment about possibility of generating some random data to test with, instead).